code/modules/mob/living/carbon/human/say.dm

```diff
@@ -30,8 +30,9 @@
 	// how do species that don't breathe talk? magic, that's what.
 	if(!HAS_TRAIT_FROM(src, TRAIT_NOBREATH, SPECIES_TRAIT) && !getorganslot(ORGAN_SLOT_LUNGS))
 		return FALSE
-	if(!dna?.species.speak_no_tongue && !getorganslot(ORGAN_SLOT_TONGUE))  // candy-etherware
-		return FALSE
+	if(dna?.species && !dna?.species.speak_no_tongue)
+		if(!getorganslot(ORGAN_SLOT_TONGUE))  // candy-etherware
```

Comment:
I don't really see the purpose of crediting a tiny addition like this. If you write a whole file, I suppose adding a little comment saying who the author is is fine, but slamming your name onto every piece of code you write is just comes off a tiny bit self-absorbed

Reply:
its not every piece of code -.-

Reply:
I've been reviewing your PRs heavily and almost all of them have some type of useless comment that exists to credit you

No other contributors do this.